x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -314,9 +314,10 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       getNavLinks$: () => navLinks$,
       setIsSidebarEnabled: (isSidebarEnabled: boolean) =>
         this.isSidebarEnabled$.next(isSidebarEnabled),
-      setGetStartedPage: (getStartedComponent: GetStartedComponent) => {
-        this.getStartedComponent = getStartedComponent;
+      setGetStartedPage: (getStartedComponent) => {
+        this.getStartedComponent$.next(getStartedComponent);
       },
+      getStorage: () => this.storage,
```

Comment:
Do we really need to use the same storage in the serverless plugin? 
Can't we just create another instance (`storage = new Storage(localStorage);`) on the other side?